spec/migrations/20181001131632_add_conversion_host_id_to_miq_request_tasks_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_migration
+
+describe AddConversionHostIdToMiqRequestTasks do
+  let(:task_stub) { migration_stub(:MiqRequestTask) }
+  let(:host_stub) { migration_stub(:Host) }
+  let(:conversion_host_stub) { migration_stub(:ConversionHost) }
+
+  migration_context :up do
+    it "when host doesn't exist" do
+      host = host_stub.create!
+      conversion_host_id = host.id
+      task = task_stub.create!(
+        :type    => 'ServiceTemplateTransformationPlanTask',
+        :options => { :dummy_key => 'dummy_value', :transformation_host_id => conversion_host_id }
+      )
+      host.destroy!
+
+      migrate
+      task.reload
+
+      expect(task.options).to eq(:dummy_key => 'dummy_value')
+      expect(AddConversionHostIdToMiqRequestTasks::ConversionHost.find_by(:resource_id => conversion_host_id)).to be_nil
```

Comment:
I think this should be `conversion_host_stub.find_by...` right?

Reply:
Done